native/common/include/jpype.h

```diff
@@ -166,4 +169,42 @@ class JPField;
 #include "jp_proxy.h"
 #include "jp_monitor.h"
 
+template <typename array_t,  typename ptr_t>
```

Comment:
Since you love whitespace changes, please delete this double space :)

Reply:
:+1: for the template pattern. Shouldn't this go to a separate header, which is specialized to arrays? If you object, please do not justify it - I trust that it you put it there for a good reason.

Reply:
I am planning a mass white space pull request, but for now I am in the tenuous position that I have backported 1/3 of JPype 0.8 into 0.7.  I am going to need to forward port the stuff that I did here back into 0.8 so I can finish up that development.  Thus I have been trying (without much success) to convince the editor to stop doing stray whitespace changes.

The template pattern can be separate if you would like.    This was the lone survivor of primitive header massacre that started that started the numpy removal process.

Reply:
Done.